custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -631,6 +673,8 @@ func (t *Translator) metricKey(timeSeries *stackdriver.TimeSeries) (string, erro
 		switch timeSeries.Resource.Type {
 		case "k8s_pod":
 			return timeSeries.Resource.Labels["namespace_name"] + ":" + timeSeries.Resource.Labels["pod_name"], nil
+		case "k8s_container":
+			return timeSeries.Resource.Labels["namespace_name"] + ":" + timeSeries.Resource.Labels["pod_name"], nil
```

Comment:
Why you selected such key that collides with pod? Would be good to document this decision by adding comment?